server/app/services/reporting/ReportingService.java

```diff
@@ -193,7 +193,7 @@ private ImmutableList<ApplicationSubmissionsStat> monthlySubmissionsAggregated(
                 aggregator = aggregators.get(stat.timestamp().get());
               } else {
                 aggregator =
-                    new ApplicationSubmissionsStat.Aggregator("All", stat.timestamp().get());
+                    new ApplicationSubmissionsStat.Aggregator("All", "All", stat.timestamp().get());
```

Comment:
question (blocking): Does the second `All` need to be localized?

Reply:
No, for reporting, we always use en_US by default.